### ISSUE TITLE
scapy does not handle windows interfaces with newlines in the name

### ISSUE THREAD

**Issue by nmsl1993** [secdev/scapy#781]
```
Traceback (most recent call last):
    from scapy.all import UDP, IP, Raw
  File "c:\python27\lib\site-packages\scapy\all.py", line 16, in <module>
    from scapy.arch import *
  File "c:\python27\lib\site-packages\scapy\arch\__init__.py", line 83, in <modu
le>
    from scapy.arch.windows import *
  File "c:\python27\lib\site-packages\scapy\arch\windows\__init__.py", line 318,
 in <module>
    IFACES.load_from_powershell()
  File "c:\python27\lib\site-packages\scapy\arch\windows\__init__.py", line 270,
 in load_from_powershell
    for i in get_windows_if_list():
  File "c:\python27\lib\site-packages\scapy\arch\windows\__init__.py", line 206,
 in get_windows_if_list
    for line in query)
  File "c:\python27\lib\site-packages\scapy\arch\windows\__init__.py", line 205,
 in <genexpr>
    (dict(zip(['name', 'win_index', 'description', 'guid', 'mac'], line))
  File "c:\python27\lib\site-packages\scapy\arch\windows\__init__.py", line 53,
in _exec_query_ps
    l.append(line.split(':', 1)[1].strip())
IndexError: list index out of range
```

Did more digging:
turns out the issue is that I have a network interface with a new line in the name. This causes an index error in windows.

**Comment by p-l-** [comment]
Hi,

Thanks for reporting this. Please next time be careful with the text formatting, I had to edit your post so that we could read it.

**Comment by p-l-** [comment]
Also, we need your exact environment (OS, Scapy & Python versions) to help you. Can you provide them?

**Comment by gpotter2** [comment]
Could you also give us the content of (by starting a `powershell.exe`):
```
Get-WmiObject Win32_NetworkAdapter | select 'Name', 'InterfaceIndex', 'InterfaceDescription', 'GUID', 'MacAddress' | fl
```

**Comment by handras** [comment]
This is solved in a #400
